go/felix/rules/endpoints.go

```diff
@@ -31,9 +31,47 @@ func (r *DefaultRuleRenderer) WorkloadEndpointToIptablesChains(epID *proto.Workl
 		WorkloadFromEndpointPfx,
 		"",
 		"",
+		untrackedDisabled,
 	)
 }
 
+func (r *DefaultRuleRenderer) HostEndpointToFilterChains(ifaceName string, endpoint *proto.HostEndpoint) []*Chain {
+	return r.endpointToIptablesChains(
+		endpoint.Tiers,
+		endpoint.ProfileIds,
+		ifaceName,
+		PolicyOutboundPfx,
+		PolicyInboundPfx,
+		HostToEndpointPfx,
+		HostFromEndpointPfx,
+		ChainFailsafeOut,
+		ChainFailsafeIn,
+		untrackedDisabled,
```

Comment:
Swapped for a `chainType` enum, which makes it more clear.  I wanted to avoid a string of `bool` arguments, which can be hard to marry up.